src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -634,6 +641,13 @@ protected function initializeContainer()
         try {
             $container = null;
             $container = $this->buildContainer();
+            if ($asFiles) {
```

Comment:
always true here - but why is this needed? it's not right now

Reply:
Same answer, I will add a new param